### ISSUE TITLE
Service Implementation Leads to Contains Unknown Lookup  Error

### ISSUE THREAD

**Issue by nesl9** [abaplint/abaplint#1799]
I created a service and the structure referenced for the entity and entitySet seem to cause the error below. 

Auto generated service implementation for MPC
```
CLASS zcl_customer_mpc DEFINITION
  PUBLIC
  INHERITING FROM /iwbep/cl_mgw_push_abs_model
  CREATE PUBLIC .

  PUBLIC SECTION.

    TYPES:
       ts_customer TYPE zs_customer. .
    TYPES:
  tt_customer TYPE STANDARD TABLE OF ts_customer. .
```

Leads to error:
`Type "TS_CUSTOMER" contains unknown: ZS_CUSTOMER not found, lookup, ZCL_CUSTOMER_MPC.CLAS.ABAP`

ZS_CUSTOMER is globally defined in the ABAP Dictionary.

A similar issue occurs with VBAK when referencing it in a data type, in this case "lookupDomain" as opposed to "lookup" 
`DATA: t_vbak      TYPE HASHED TABLE OF vbak WITH UNIQUE KEY vbeln.`

Leads to error:
`Variable "LT_VBAK" contains unknown: ZUONR, lookupDomain (unknown_types)`

**Comment by larshp** [comment]
ZS_CUSTOMER: is the object in the repository? if so I guess this can be reproduced with some dummy code in a public repository?

VBAK: which setup are you running? VBAK would typically be unknown to abaplint, so it will not know that there is a field called "ZUONR"? unless you have built custom deps or something?

**Comment by larshp** [comment]
VBAK example in the default setup, 

```abap
DATA lt_vbak TYPE HASHED TABLE OF vbak WITH UNIQUE KEY vbeln.
LOOP AT lt_vbak ASSIGNING FIELD-SYMBOL(<ls_vbak>).
  WRITE / <ls_vbak>-its_voided.
ENDLOOP.
```

![image](https://user-images.githubusercontent.com/5888506/109047053-b616ef00-76d5-11eb-868a-3e803ddb584c.png)

**Comment by nesl9** [comment]
ZS_CUSTOMER: I can create a public repo. 

VBAK: Got it!

**Comment by larshp** [comment]
the `errorNamespace` in abaplint.json controls which objects are trusted(voided) and which abaplint will report errors for

however, if the object is defined in one of the repositories under `dependencies` in abaplint.json, it will be fully checked

abaplint requires knowledge about the superclass and implemented interfaces in order to be able to do a somewhat qualified syntax check of the code

**Comment by larshp** [comment]
or did you get any of the errors via https://github.com/abaplint/abaplint-sci-client ?

**Comment by nesl9** [comment]
Yes, I followed the installation from abaplint-sci-client. Running the abaplint server locally using Docker. Running SAP ERP 6.0 Ehp8  with ABAP 7.5 also in a container.  The error comes up via Code Inspector in abapGit. I have a slightly customized abaplint.json (i.e. different data type naming convention). If it's helpful I can create the public repo if not I can look into the dependencies in abaplint.json
